rsqueakvm/model/pointers.py

```diff
@@ -264,3 +232,60 @@ def clone(self, space):
                                     len(my_pointers))
         w_result.store_all(space, my_pointers)
         return w_result
+
+
+# when changing this constant, also change the read and __write__ methods in
+# IntMapStorageNode in storage.py (or generalize those methods to compile with
+# getattrs)
+_NUMBER_OF_INT_FIELDS = 3
+_NUMBER_OF_INLINE_FIELDS = 2
+class W_FixedPointersObject(W_PointersObject):
```

Comment:
Right now the number of inline fields here is truly random, i.e., determined by a fair dice roll. Not sure if we want to tweak that or have multiple FixedPointersObjects or what.